docs/platforms/javascript/common/best-practices/micro-frontends.mdx

```diff
@@ -21,22 +21,22 @@ keywords:
     "module metadata",
   ]
 ---
+<Alert>
+In order for the Sentry JavaScript SDK to work with your micro frontends, you'll need to either set up [module federation](https://module-federation.github.io/) or make sure that all your micro frontends are using the same version of the Sentry SDK.
```

Comment:
I'd reword this because to my understanding (also double checked with the team), module federation has nothing to do with this requirement. Module federation is just one way of setting up an MFE architecture but the requirement still applies. 

I also added the package part to the sentence because this is only important if individual MFEs actually import from a @sentry package. There are also MFE<>Sentry use cases where only the host app cares about sentry and all errors are caught automatically and routed to the respective projects as described below.

```suggestion
In order for the Sentry JavaScript SDK to work with your micro frontends, make sure that all micro frontends that import from an `@sentry/*` package are using the same version of the Sentry SDK.
```

Reply:
Thanks for checking!